docs/plugins/02-default-plugins.md

```diff
@@ -106,6 +106,18 @@ View the documentation with `:help luasnip`
 
 LuaSnip uses friendly-snippets as it's snippet library.
 
+```lua
+-- Beside defining your own snippets you can also load snippets from "vscode-like" packages
+require("luasnip/loaders/from_vscode").load({ paths = { "./my-snippets" } })
+-- Example on how to add your own
+local ls = require("luasnip")
+ls.snippets = {
+	javascript = {
+		ls.parser.parse_snippet("cv", 'console.log("${1:object} ", ${1})'),
+	},
+}
```

Comment:
This should probably go higher, and specifically mention that this is the lua syntax.